app/views/email/CaptureEmailView.scala.html

```diff
@@ -19,10 +19,11 @@
 @import forms.EmailForm._
 @import views.html.templates.inputs.Text
 @import views.html.templates.errors.ErrorSummary
+@import play.api.mvc.Call
 
 @this(mainTemplate: MainTemplate, form: FormWithCSRF, errorSummary: ErrorSummary, text: Text)
 
-@(emailForm: Form[String], emailNotChangedError: Boolean, currentEmail: String)(implicit user: User[_], messages: Messages, appConfig: AppConfig)
+@(emailForm: Form[String], emailNotChangedError: Boolean, currentEmail: String, submitFormAction: Call)(implicit user: User[_], messages: Messages, appConfig: AppConfig)
```

Comment:
The "Back" link on this page (line 36) is going back into BTA account details which would be incorrect for this journey. The previous page for this journey would be the show action in `AddEmailAddressController`. Looks like you might need to pass another parameter into this view to support that.